src/main/java/oss/fosslight/controller/OssController.java

```diff
@@ -526,9 +526,6 @@ public String copy(@PathVariable String ossId, HttpServletRequest req, HttpServl
 		if(ossMailInfo.getOssNicknames() != null) {
 			ossMailInfo.setOssNickname(CommonFunction.arrayToString(ossMailInfo.getOssNicknames(), "<br>"));	
 		}
-		
-		// 삭제처리
-		ossService.deleteOssMaster(ossMaster);
 
 		CoCodeManager.getInstance().refreshOssInfo();

```

Comment:
@yugeeklab , @hubert.you,
Please review whether the following row should be called after deleteOssMaster.
`CoCodeManager.getInstance().refreshOssInfo();`

Reply:
`CoCodeManager.getInstance().refreshOssInfo(); ` must be called after `ossService.deleteOssMaster()`. 
@FOSSLight-dev will fix it later.